docs/UserGuide.md

```diff
@@ -699,19 +759,19 @@ If a parameter is expected only once in the command, but you specified it multip
 [keyboard commands](#summary-of-keyboard-commands) which when pressed, will fill in the command keyword for you.
```

Comment:
Actually, would it be good to put the table here? Since we are describing the features.

Reply:
Sounds good.